tests/controllers/UsersControllerTest.php

```diff
@@ -226,4 +226,192 @@ public function testThatUserDataIsNotDoubleEscapedOnUserEdit()
 
         $controller->updateUser($request, $db);
     }
+
+    /**
+     * Ensures that if the setTrusted method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
```

Comment:
Tiny but this line does nothing. With or without it the test is run due to the prefix.  Can it be one of the other (prefix or annotation)